src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+@import 'var';
+
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: $background-white-color;
+  max-width: 198px;
+  margin-bottom: 500px !important;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    & img {
+      width: 100%;
+    }
+  }
+
+  &__title {
+    color: rgb(6, 11, 53);
```

Comment:
Consider defining `rgb(6, 11, 53)` as an SCSS variable to maintain consistency across your stylesheets.